### DISCUSSION TITLE
What is this strange keymap? It prevent me using <space>w instantly, I have to wait 100ms and can't delete the keymap

### DISCUSSION THREAD

**Discussion by xzbdmw** [LazyVim/LazyVim#2478] [Q&A] [answered]
This is original lazyvim
<img width="1173" alt="image" src="https://github.com/LazyVim/LazyVim/assets/97848247/db84509b-dea5-4f20-93f1-13ceb16a7db4">

**Comment by Neurognostic** [comment, marked as answer]
Looks like it comes from here:

https://github.com/folke/which-key.nvim/blob/4433e5ec9a507e5097571ed55c02ea9658fb268a/lua/which-key/keys.lua#L6

And its purpose is explained here:

https://github.com/folke/which-key.nvim/issues/299#issuecomment-1239618887
